src/composables/accounts.ts

```diff
@@ -26,6 +26,8 @@ import { useStorageRef } from './storageRef';
 
 let isInitialized = false;
 
+const isAccountsRestored = ref(false);
```

Comment:
Do you think this name is OK? Or maybe `areAccountsRestored`?

Reply:
`areAccountsRestored` is better.